patches/chrome-browser-importer-importer_list.cc.patch

```diff
@@ -26,15 +26,6 @@ index 3bf47fa746e2a417f89201f3e3800d8639f2e323..4152d44b23d1b5b1e909e351da31f2a5
  namespace {
  
  #if defined(OS_WIN)
-@@ -119,7 +126,7 @@ void DetectFirefoxProfiles(const std::string locale,
-     firefox.app_path = app_path;
-   firefox.services_supported = importer::HISTORY | importer::FAVORITES |
-                                importer::PASSWORDS | importer::SEARCH_ENGINES |
--                               importer::AUTOFILL_FORM_DATA;
-+                               importer::AUTOFILL_FORM_DATA | importer::COOKIES;
```

Comment:
cc @darkdh

Reply:
I know you didn't work on this for brave-core, but you have experience with it from muon

Reply:
@bbondy why removing this? This is required for import cookies option from Firefox

Reply:
<img width="284" alt="screen shot 2018-08-24 at 1 19 27 pm" src="https://user-images.githubusercontent.com/11330831/44606028-6ff91180-a7a0-11e8-985e-49c214fb2039.png">

Reply:
Thanks for finding, this was an unintentional miss when applying the patch manually. Btw this patch is really big and we should consider reformatting some of the code to make it easier to rebase.


```
diff --git a/chrome/browser/importer/importer_list.cc b/chrome/browser/importer/importer_list.cc
index c8b1156c1be3d4b31c266b723e3a3ea9e50c73e8..a0dfd62cb0a9aad31573604d4bd884ca4a9cb7e6 100644
--- a/chrome/browser/importer/importer_list.cc
+++ b/chrome/browser/importer/importer_list.cc
@@ -7,9 +7,13 @@
 #include <stdint.h>
 
 #include "base/bind.h"
+#include "base/strings/utf_string_conversions.h"
 #include "base/task/post_task.h"
 #include "base/task/task_traits.h"
 #include "base/threading/thread_restrictions.h"
+#include "base/values.h"
+#include "brave/common/importer/brave_importer_utils.h"
+#include "brave/common/importer/chrome_importer_utils.h"
 #include "build/build_config.h"
 #include "chrome/browser/shell_integration.h"
 #include "chrome/common/importer/firefox_importer_utils.h"
@@ -29,6 +33,9 @@
 #include "chrome/common/importer/edge_importer_utils_win.h"
 #endif
 
+void DetectChromeProfiles(std::vector<importer::SourceProfile>* profiles);
+void DetectBraveProfiles(std::vector<importer::SourceProfile>* profiles);
+
 namespace {
 
 #if defined(OS_WIN)
@@ -136,21 +143,37 @@ std::vector<importer::SourceProfile> DetectSourceProfilesWorker(
 #if defined(OS_WIN)
   if (shell_integration::IsFirefoxDefaultBrowser()) {
     DetectFirefoxProfiles(locale, &profiles);
+    DetectBraveProfiles(&profiles);
     DetectBuiltinWindowsProfiles(&profiles);
+    DetectChromeProfiles(&profiles);
   } else {
     DetectBuiltinWindowsProfiles(&profiles);
+    DetectBraveProfiles(&profiles);
     DetectFirefoxProfiles(locale, &profiles);
+    DetectChromeProfiles(&profiles);
   }
 #elif defined(OS_MACOSX)
   if (shell_integration::IsFirefoxDefaultBrowser()) {
     DetectFirefoxProfiles(locale, &profiles);
+    DetectBraveProfiles(&profiles);
     DetectSafariProfiles(&profiles);
+    DetectChromeProfiles(&profiles);
   } else {
     DetectSafariProfiles(&profiles);
+    DetectBraveProfiles(&profiles);
     DetectFirefoxProfiles(locale, &profiles);
+    DetectChromeProfiles(&profiles);
   }
 #else
-  DetectFirefoxProfiles(locale, &profiles);
+  if (shell_integration::IsFirefoxDefaultBrowser()) {
+    DetectFirefoxProfiles(locale, &profiles);
+    DetectBraveProfiles(&profiles);
+    DetectChromeProfiles(&profiles);
+  } else {
+    DetectBraveProfiles(&profiles);
+    DetectChromeProfiles(&profiles);
+    DetectFirefoxProfiles(locale, &profiles);
+  }
 #endif
   if (include_interactive_profiles) {
     importer::SourceProfile bookmarks_profile;
```

Reply:
yeah, we have a main issue for minimizing importer patch https://github.com/brave/brave-browser/issues/511